dapps/marketplace/src/components/Reviews.js

```diff
@@ -13,6 +13,8 @@ import QueryError from 'components/QueryError'
 import query from 'queries/Reviews'
 import EthAddress from './EthAddress'
 
+import LoadingSpinner from 'components/LoadingSpinner'
```

Comment:
It may just be me, but I like to keep all the sibling `import`s grouped together, aka this one on line 11, perhaps. 😁

Reply:
Definitely not just you, I'll fix that!